Formula/premake.rb

```diff
@@ -23,7 +23,14 @@ class Premake < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "f18473b6bf3eb65b2b8e36f35e2166bfe6f859abae5e63806f7bcb5ec675f2bf"
   end
 
+  on_linux do
+    depends_on "util-linux" # for uuid
+  end
+
   def install
+    # Workaround for Xcode 14.3
+    ENV.append_to_cflags "-Wno-implicit-function-declaration" if DevelopmentTools.clang_build_version >= 1403
```

Comment:
Can we use the patch at https://github.com/nih-at/libzip/commit/70629b26c0ad0a709c444921574f49454850dea0 instead?

Reply:
only that patch is not enough

```
../../contrib/libzip/zip_close.c:621:2: error: call to undeclared function 'close'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
        close(tfd);
        ^
zutil.c
```

Reply:
Looks like they don't pass `-DHAVE_UNISTD_H` correctly, so we might need to do that ourselves as well until upstream fix it.